azure-pipelines-official.yml

```diff
@@ -82,11 +82,11 @@ stages:
     demands: ImageOverride -equals Build.Windows.Amd64.VS2022
 
   jobs:
-  - ${{ if eq(variables['Build.SourceBranch'], 'refs/heads/release/dev17.2') }}:
+  - ${{ if eq(variables['Build.SourceBranch'], 'refs/heads/release/dev17.3') }}:
     - template: /eng/common/templates/job/onelocbuild.yml
       parameters:
         MirrorRepo: roslyn
-        MirrorBranch: release/dev17.2
+        MirrorBranch: release/dev17.3
```

Comment:
at what point do we update this in general?  Should it be done every time we create a new release branch?

Reply:
We localize one branch at a time and it requires that this stay in sync with internal localization configuration. I believe this change is typically requested after we've snapped for the last preview that will contain new features.